src/features/settings/components/InventorySetExportSection/InventorySetExportSection.module.css

```
@@ -0,0 +1,118 @@
+  font-weight: 500;
+  color: var(--color-text);
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-sm);
+}
+
+.activeBadge {
+  font-size: var(--font-size-xs);
+  font-weight: normal;
+  color: var(--color-primary);
+  background-color: var(--color-primary-light, rgba(59, 130, 246, 0.1));
```

Stylelint (17.2.0)

[error] Expected "0.1" to be "10%" (alpha-value-notation)

(alpha-value-notation)